_projects/2018/poliastro/benchmark-propagators.md

```diff
@@ -0,0 +1,73 @@
+---
+name: Benchmark several propagators
```

Comment:
Maybe, `Benchmark orbit propagators` is more attractive?

Reply:
Totally!